src/modules/test/ManagementModule.cpp

```diff
@@ -0,0 +1,517 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this file a duplicate of `src/platform/ManagementModule.cpp` ?

Reply:
Yes. Discussed today. Bad idea to link to original directly.